pylib/gyp/generator/compile_commands_json.py

```diff
@@ -109,7 +109,7 @@ def GenerateOutput(target_list, target_dicts, data, params):
         AddCommandsForTarget(cwd, target, params, per_config_commands)
 
     try:
-        output_dir = params["options"].generator_output
+        output_dir = params["options"].generator_output or os.getcwd()
```

Comment:
Can you add a comment to here to why `or os.getcwd()` is needed?

Reply:
I believe that the comment on 112 already explains why.